### PR TITLE
updates to `HEYWHATSTHAT_ALTS` handling

### PR DESCRIPTION
**Pull request by kx1t**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -184,7 +184,7 @@ This container accepts HTTP connections on TCP port `80` by default. You can cha
 | `MLATPORT` | Optional. TCP port number of an MLAT provider (`mlat-client`) | 30105 |
 | `TZ` | Optional. Your local timezone in [TZ-database-name](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) format | |
 | `HEYWHATSTHAT_PANORAMA_ID` | Optional. Your `heywhatsthat.com` panorama ID. See <https://github.com/wiedehopf/tar1090#heywhatsthatcom-range-outline>. | |
-| `HEYWHATSTHAT_ALTS` | Optional. Comma separated altitudes for multiple outlines. Use no units or `ft` for feet, or `m` for meters | `12192m` (=40000 ft) |
+| `HEYWHATSTHAT_ALTS` | Optional. Comma separated altitudes for multiple outlines. Use no units or `ft` for feet, `m` for meters, or `km` for kilometers. Only integer numbers are accepted, no decimals please | `12192m` (=40000 ft) |
```
Comment (dirkhh) [inline README.md:187]: do you really think that people will want to write this in km?
That seems a bit "over-engineered to me" 😇

File: rootfs/etc/cont-init.d/06-range-outline [hunk 2]
```diff
@@ -3,6 +3,25 @@
...
+
+    # convert all elements to meters
+    readarray -td, alts_array <<< "${HEYWHATSTHAT_ALTS:-12192m}"
+    for i in "${!alts_array[@]}"
+    do
+        alts_array[i]="${alts_array[i]//$'\n'/}"
+        if [[ ${alts_array[i]: -1} == "m" ]]; then
+            alts_array[i]="${alts_array[i]:0:-1}"
+        elif [[ ${alts_array[i]: -2} == "ft" ]]; then
+            alts_array[i]="$(bc -l <<< "scale=0; ${alts_array[i]:0:-2}/3.281")"
```
Comment (dirkhh) [inline rootfs/etc/cont-init.d/06-range-outline:25]: do we know that `bc` is always installed?

**Review by dirkhh — COMMENTED** [review]
admittedly, this is at the very edge of my bash knowledge.
I think this should work (I had to google one of the constructs, TBH). But I am far from an expert at this level.

**Comment by fredclausen** [comment]
Also, it would be worth noting what the default unit is, which appears to be meters?

**Pull request merged**